test/e2e/cli_test.go

```diff
@@ -1011,7 +1011,6 @@ func (s *CLIWithServerSuite) TestTemplateLevelSemaphore() {
 }
 
 func (s *CLISuite) TestRetryOmit() {
-	s.T().Skip("flaky - see https://github.com/argoproj/argo/issues/3540")
```

Comment:
Test seems to always pass locally. Opening PR to see if I can get a fail in the CI.